common/version.go

```diff
@@ -0,0 +1,20 @@
+// Package common implements common things for Oasis Core Rosetta Gateway.
+package common
+
+import (
+	"runtime"
+	"strings"
+)
+
+var (
+	// SoftwareVersion represents the Oasis Core Rosetta Gateway's version and
+	// should be set by the linker.
+	SoftwareVersion = "0.0.0-unset"
+
+	// RosettaAPIVersion represents the Rosetta API version with which the
+	// Oasis Core Rosetta Gateway is guaranteed to be compatible with.
+	RosettaAPIVersion = "1.4.1"
```

Comment:
This shouldn't be hard-coded in the future, see: #129.

Reply:
You should also make https://github.com/oasisprotocol/oasis-core-rosetta-gateway/blob/master/services/network.go#L135 use the RosettaAPIVersion specified here :)

Reply:
> You should also make https://github.com/oasisprotocol/oasis-core-rosetta-gateway/blob/master/services/network.go#L135 use the RosettaAPIVersion specified here :)

Good catch, fixed.